src/main/scala/io/citrine/lolo/util/Async.scala

```diff
@@ -9,6 +9,7 @@ object Async {
     * Check the thread's interrupted status, because it could stop
     */
   def canStop(): Unit = {
+    Thread.sleep(1) // sleep for 1 ms to yield
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/lang/Thread.html#yield()

> "It is rarely appropriate to use this method"

Sleep was suggested as being preferred and far more robust.